vividus-plugin-web-app/src/main/java/org/vividus/bdd/steps/ui/web/WaitSteps.java

```diff
@@ -103,6 +103,35 @@ public boolean waitDurationWithPollingDurationTillElementAppears(Duration durati
                 expectedSearchActionsConditions.visibilityOfAllElementsLocatedBy(locator)).isWaitPassed();
     }
 
+    /**
+     * Waits <b>duration</b> with <b>pollingDuration</b> until <b>an element</b> by the specified <b>locator</b>
+     * disappearance in the specified search context
+     * <p>
+     * Actions performed at this step:
+     * </p>
+     * <ul>
+     * <li><i>Finds</i> an element by the specified <b>locator</b> within search context
+     * <li><i>Waits</i> <b>duration</b> with <b>pollingDuration</b> until this element becomes not visible
+     * </ul>
+     * <br>Example:<br>
+     * <code>When I wait 'PT30S' with 'PT10S' polling until an element located by By.id(text) disappears</code>
+     * - wait until all elements with id=text for 30 seconds, polling every 10 seconds
+     *
+     * @param duration        Total waiting time for an element becomes not visible according to
+     *                        <a href="https://en.wikipedia.org/wiki/ISO_8601">ISO 8601</a> standard
+     * @param pollingDuration Defines the timeout between attempts according to
+     *                        <a href="https://en.wikipedia.org/wiki/ISO_8601">ISO 8601</a> standard
+     * @param locator         Locator to search for elements
+     * @return True if element disappears, otherwise false
+     */
+    @When("I wait '$duration' with '$pollingDuration' polling until an element located by $locator disappears")
```

Comment:
What do you think about making this step generic like
When I wait '$duration' with '$pollingDuration' polling until element located `$locator` becomes $state
this would reduce "duplicate" code base and make those steps as composites